magento/pwa-studio
# [Doc] minor doc site updates

## Description

Makes minor changes to the doc site:

- Add 6.0.1 to the compatibility matrix
- Update the content on the front page
- Remove the message banner
- Add Lars to the contributor page

## Related Issue

## Acceptance 

@awilcoxa 

### Verification Stakeholders

@dpatil-magento 

### Specification



### Verification Steps

1. Build doc site
1. Verify no message banner appears on the site
1. Verify updated cards on the front page with working links
1. Verify Lars appears on the `/technologies/contribute/` page
1. Verify correct data on the `/technologies/magento-compatibility/` page

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
